src/profile-v2/CertificateCard.jsx

```diff
@@ -37,24 +40,47 @@ const CertificateCard = ({
           className="certificate-type-illustration"
           style={{ backgroundImage: `url(${certificateIllustration})` }}
         />
-        <div className="d-flex flex-column position-relative p-0 width-19625rem">
+        <div className={classNames(
+          'd-flex flex-column position-relative p-0',
+          { 'max-width-19rem': isMobileView },
+          { 'width-19625rem': !isMobileView },
+        )}
+        >
           <div className="w-100 color-black">
-            <p className="small mb-0 font-weight-normal">
+            <p className={classNames([
+              'mb-0 font-weight-normal',
+              isMobileView ? 'x-small' : 'small',
+            ])}
+            >
```

Comment:
follow this pattern for better readability 
<p className={classNames('mb-0 font-weight-normal', {
  'x-small': isMobileView,
  'small': !isMobileView,
})}>